oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -463,7 +463,6 @@ static int mca_spml_ucx_component_fini(void)
     volatile int fenced = 0;
     int i;
     int ret = OSHMEM_SUCCESS;
-    mca_spml_ucx_ctx_t *ctx;
```

Comment:
Does it have to do anything with the fix?

Reply:
It's just another compilation warning coming from ucx component thought I will handle that one too